patterns/roks-quickstart/variables.tf

```diff
@@ -0,0 +1,269 @@
+##############################################################################
+# Account Variables
+##############################################################################
+
+variable "ibmcloud_api_key" {
+  description = "The IBM Cloud platform API key needed to deploy IAM enabled resources."
+  type        = string
+  sensitive   = true
+}
+
+variable "prefix" {
+  description = "A unique identifier for resources. Must begin with a lowercase letter and end with a lowerccase letter or number. This prefix will be prepended to any resources provisioned by this template. Prefixes must be 16 or fewer characters."
+  type        = string
+  default     = "land-zone-vsi-qs"
+
+  validation {
+    error_message = "Prefix must begin with a lowercase letter and contain only lowercase letters, numbers, and - characters. Prefixes must end with a lowercase letter or number and be 16 or fewer characters."
+    condition     = can(regex("^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$", var.prefix)) && length(var.prefix) <= 16
+  }
+}
+
+variable "region" {
+  description = "Region where VPC will be created. To find your VPC region, use `ibmcloud is regions` command to find available regions."
+  type        = string
+  default     = "us-south"
+}
+
+variable "resource_tags" {
+  type        = list(string)
+  description = "Optional list of tags to be added to created resources"
+  default     = []
+}
+
+variable "override_json_string" {
+  description = "Override default values with custom JSON. Any value here other than an empty string will override all other configuration changes."
+  type        = string
+  default     = <<EOF
+{
+   "atracker": {
+      "collector_bucket_name": "",
+      "receive_global_events": false,
+      "resource_group": "",
+      "add_route": false
+   },
+   "clusters": [
+      {
+         "boot_volume_crk_name": "slz-vsi-volume-key",
+         "cos_name": "cos",
+         "kms_config": null,
+         "kube_type": "openshift",
+         "kube_version": "4.14_openshift",
```

Comment:
I think we should put this variable at the top level of the DA, it is going to be a common case for people to change it.
Same for the machine type.

Reply:
From an implementation perspective, perhaps we could make the override json a template here. So instead of 
"kube_version": "4.14_openshift",
we would have "kube_version": "${version}" with version being the input variable.

The quick start code would replace the version in the override json and pass it to the ../roks/module.